### ISSUE TITLE
Question about trainable/frozen param detail

### ISSUE THREAD

**Issue by MilkClouds** [NVlabs/vla0#12]
Hi, I have some question about your awesome work:

- MLLM is composed with vision encoder, connector and LLM. especially on LLM we have (1) embedding (2) others (3) and lm_head. Among this which component you freeze and which is not? is it full fine-tuning?
- on my understanding paper states that there was no LoRA, it's full fine-tuning, but I want to ask you and ensure whether it is right.
- I'm curious about any more missing details about training setting. e.g. presence of weight decay
